postgresql-containers/build/postgres/Dockerfile

```diff
@@ -72,16 +89,21 @@ RUN set -ex; \
         gettext \
         hostname \
         perl \
+        tar \
         libedit \
         openssh \
         openssh-server \
         openssh-clients \
-        procps-ng; \
+        procps-ng \
+        bzip2 \
+        lz4; \
     sed -i 's/enabled=1/enabled=0/' /etc/yum.repos.d/epel*.repo; \
     sed -i 's/install_langs C.utf8/install_langs all/' /etc/rpm/macros.image-language-conf; \
     microdnf -y module disable llvm-toolset; \
     microdnf -y install \
+        systemd \
```

Comment:
purely curious, why systemd was required?

Reply:
I don't know, I just copied it from the pgbackrest dockerfile